#!/usr/bin/env python
PACKAGE = "rb1"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

method_enum = gen.enum([ gen.const("Euler",      	int_t,  0,  "Euler integration method"),
                       	 gen.const("RungeKutta",   int_t,  1,  "RungeKutta integration method")],
                      "An enum to set odometry integration method")

gen.add("integMethod", int_t, 0, "An integration method parameter which is edited via an enum", 0, 0, 1, edit_method = method_enum)

exit(gen.generate(PACKAGE, "rb1_odometry", "parameters"))